react/store/createReturnRequest/components/ItemsDetails.tsx

```diff
@@ -44,7 +44,25 @@ export const ItemsDetails = (props: Props) => {
     hints: { phone },
   } = useRuntime()
 
+  const CSS_HANDLES_PHONE = [
+    'cardWrapper',
+    'productImageWrapper',
+    'productImage',
+    'productDetailsWrapper',
+    'productText',
+    'quantityWrapper',
+    'quantityKey',
+    'quantityValue',
+    'availableToReturnWrapper',
+    'availableToReturnKey',
+    'availableToReturnValue',
+    'quantitySelectorWrapper',
+    'reasonWrapper',
+    'conditionWrapper',
+  ] as const
+
   const handles = useCssHandles(CSS_HANDLES)
+  const handlesPhone = useCssHandles(CSS_HANDLES_PHONE)
```

Comment:
There is no need to create a separate array for the CSS Handles.
Additionally, I see that some of this classes are already declared and are used in the table. This could create conflicts when working with them.